Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35145&attempt_id=c327a471-899b-46e1-a44c-63f606fc9614)

<details><summary>Partial log (click to expand)</summary><p>

```
patching sources
configuring
no configure script, doing nothing
building
build flags: SHELL=/nix/store/19cc89zq5n68ibr53izy4zgcr5qg2rza-bash-4.4-p12/bin/bash
gcc -Wall -Wextra cue2pops.c -o cue2pops
/nix/store/19cc89zq5n68ibr53izy4zgcr5qg2rza-bash-4.4-p12/bin/bash: gcc: command not found
make: *** [Makefile:20: cue2pops] Error 127
builder for '/nix/store/z27bjirb4wahpy76cbcyvyspdqw5pxnr-cue2pops-git-2018-01-04.drv' failed with exit code 2
error: build of '/nix/store/z27bjirb4wahpy76cbcyvyspdqw5pxnr-cue2pops-git-2018-01-04.drv' failed
```
</p></details>

--- a/pkgs/tools/cd-dvd/cue2pops/default.nix
+++ b/pkgs/tools/cd-dvd/cue2pops/default.nix
@@ -14,6 +14,8 @@ stdenv.mkDerivation rec {
 
   dontConfigure = true;
 
+  makeFlags = ["CC=cc"];
+
   installPhase = ''
     install --directory --mode=755 $out/bin
     install --mode=755 cue2pops $out/bin

